class V
{
	int test()
	{
		try
		{
			return 10;	
		}
		catch (ArithmeticException ex)
		{
		}
	}
}
